estuary-cdk/estuary_cdk/pydantic_polyfill.py

```diff
@@ -0,0 +1,55 @@
+"""
```

Comment:
What can't we use pydantic v2 directly?

Reply:
We can, for connectors we write ourselves. This polyfill is required for Airbyte connectors we're wrapping, because airbyte depends on pydantic v1. The polyfill makes pydantic v1 look enough like v2 that the CDK can use it.